app/javascript/react/components/reviews/ReviewList.js

```diff
@@ -1,17 +1,19 @@
 import React from "react"
 import ReviewTile from "./ReviewTile"
 
-const ReviewList = (props) => {
-  const { reviews } = props
+const ReviewList = props => {
+  const { reviews, animal, deleteReview, currentUser } = props
 
-  let reviewList = reviews.map((review) => {
+  let reviewList = reviews.map(review => {
     return (
       <ReviewTile
         key={review.id}
         title={review.title}
         description={review.description}
         rating={review.rating}
-        user="placeholder person"
+        deleteReview={() => deleteReview(review.id, animal.id)}
```

Comment:
instead of passing a callback function, we can define a wrapper function within this map (handleDeleteReview) won't take any arguments but will call on delete review and that's where we can pass in the review.id as an argument. This mainly helps with readability and tracking for debug purposes.

Also your fetch request has animal.id already so you shouldn't need to pass it here as well.